cloud/azure/management/monitor/mode/getmetrics.pm

```diff
@@ -230,6 +235,10 @@ Set resource type (Required if resource's name is used).
 
 Set monitor metrics (Required) (Can be multiple).
 
+=item B<--metricnamespace>
```

Comment:
Tiny detail, could you name it --metric-namespace with a dash as word separator ;)